cfpb/cfgov-refresh
# Moving /your-story/ into cfgov-refresh

Moving /your-story/ into cfgov-refresh

## Additions

- Added /your-story/ template, and css to cfgov-refresh.


## Changes

- Modified `urls.py` to add code for cfgov-refresh.

## Testing

1. Visit `http://localhost:8000/your-story` 
2. Compare to https://www.consumerfinance.gov/your-story/ for differences.
3. Visit the various breakpoints (599, 900, 1200) and verify that the page works as expected.
4. Click the various buttons and links and verify the page works as expected.

## Screenshots

Mobile
<img width="394" alt="screen shot 2017-08-15 at 5 31 05 pm" src="https://user-images.githubusercontent.com/1696212/29337559-b570c9f2-81df-11e7-9994-71e5c81a42bd.png">

Tablet
<img width="646" alt="screen shot 2017-08-15 at 5 30 54 pm" src="https://user-images.githubusercontent.com/1696212/29337558-b5706d9a-81df-11e7-834f-4b63c2d65f04.png">

Desktop
<img width="1278" alt="screen shot 2017-08-15 at 5 29 47 pm" src="https://user-images.githubusercontent.com/1696212/29337560-b572f0d8-81df-11e7-8259-1f37c146b9c7.png">

## Notes

- Not many styles were needed, so I added them inline. The hero styles are already inline. 
- I didn't flag the route because the Apache redirect is in place.

## Todos

-

## Checklist
